### DISCUSSION TITLE
Navigation design : dock menu or create frames

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7805] [Help] [answered]
Is it possible to have the navigation menu docked at the top (or on the left) so it remains always visible even if we scroll long down or right ?

And / or is it possible to create frames ? I thought it was possible using container section Main/sidebar or sidebar/main but I never see a scrollbar appearing : I want the navigation **and** the red section to remain at the top even if the green section goes down very far :
<img width="958" alt="image" src="https://user-images.githubusercontent.com/28592069/190652500-2b7ef098-6f4b-4867-9551-5d9c679d5b8a.png">

**Comment by melohagan** [comment]
For me the navigation bar when positioned to the left does anchor, however when positioned at the top it does not appear to anchor.

This can be altered with a little bit of CSS in an Embed component. 

Add the embed component to your screen with the following content:

```html
<style>
.nav-wrapper {
  position: sticky !important;
  top: 0;
  left: 0;
  display: fixed !important;
}
</style>
```

**Reply by NicolasFabre81** [reply]
Sorry but it doesn't appear to work, maybe I did not manage to get understood, sorry for my english !
What I'd like to have is a vertical scrollbar in either the left section, or the right one.
I tested your code with another page, see :
<img width="954" alt="image" src="https://user-images.githubusercontent.com/28592069/192465216-aa0e5412-09f1-4c52-95a7-f7e4e1f245b1.png">

My components :
<img width="105" alt="image" src="https://user-images.githubusercontent.com/28592069/192465313-3fb9d423-27bf-4505-812d-34904f832ec3.png">

**Comment by melohagan** [comment, marked as answer]
Hey @NicolasFabre81 

I think I may understand what you want now. A section that has a separate scrollbar per section column?

Embed content:
```html
<style>
.component .container {
  overflow: scroll !important;
  height: 500px !important;
}
</style>
```

![Screenshot 2022-09-28 at 09 13 38](https://user-images.githubusercontent.com/101575380/192725906-58579a96-6ac4-417f-9235-563d50ebc772.png)

**Reply by NicolasFabre81** [reply]
YES  that's what I want !
But I don't manage to make your embed code work...
It only adds me a scrollbar on the right, and the left section is combined with the right one.
Does the place of the embed tool matter ?
<img width="635" alt="image" src="https://user-images.githubusercontent.com/28592069/192833885-f5b63af4-ed36-48c2-a2ce-f8fd2ff3461f.png">

My tools :
<img width="143" alt="image" src="https://user-images.githubusercontent.com/28592069/192834043-e0ff6172-7932-42b2-b1d2-db37c94783b0.png">

**Reply by melohagan** [reply]
The location of the embed shouldn't matter. I wonder if it's because your Section is inside a container. Maybe try:

> Embed
> Section
> > Container
> > Container

**Comment by NicolasFabre81** [comment]
As far as I know you cannot create a section outside of a container.

In fact when I was trying, at one time, I did see the correct behaviour. But then I changed something and it stopped to work, do not manage to have it working again.

If you could put here a little working test app I'm sure it'd be useful for a lot of budibasers :-)

**Reply by NicolasFabre81** [reply]
Eureka !
I just added "overflow: scroll !important;" in the custom CSS of my right container, it works !

**Reply by melohagan** [reply]
Fantastic!
